src/psyclone/tests/psyir/nodes/container_test.py

```diff
@@ -110,6 +110,7 @@ def test_container_create():
     assert result == (
         "module container_name\n"
         "  real :: tmp\n\n"
+        "  implicit none\n"
```

Comment:
Again, this won't compile. Please could you add a compilation check to this test (if poss).

Reply:
Fixed placement. But I think this one shouldn't have the compilation as we are just checking the creation. Using the Fortran output is circumstantial in this test.

Reply:
OK.